alchemist/alchemist-swingui/src/main/kotlin/it/unibo/alchemist/input/KeyboardEvents.kt

```diff
@@ -17,7 +17,9 @@ import javafx.scene.input.KeyEvent
  */
 enum class ActionOnKey {
     PRESSED,
-    RELEASED
+    RELEASED;
+
+    operator fun plus(key: KeyCode): KeyboardTriggerAction = KeyboardTriggerAction(this, key)
```

Comment:
uhmm... If understand correctly, this combines an `ActionOnKey` and a `KeyCode` to a `TriggerAction`. I'm not sure we want to use `+`, it's not a natural sum. maybe an infix function `with`?

``keyboard.setOnAction(ActionOnKey.PRESSED with it, enqueueMove)``

Or another keyword, I'm open to suggestions, I'ma bit reluctant to overload operators too much.

Reply:
That makes sense, I made the adjustment.